platforms/core-execution/persistent-cache/src/main/java/org/gradle/cache/internal/DefaultCacheCoordinator.java

```diff
@@ -321,7 +322,7 @@ public <K, V> MultiProcessSafeIndexedCache<K, V> newCache(final IndexedCachePara
                 if (decorator != null) {
                     indexedCache = decorator.decorate(cacheFile.getAbsolutePath(), parameters.getCacheName(), indexedCache, crossProcessCacheAccess, getCacheAccessWorker());
                     if (fileLock == null) {
-                        useCache(NO_OP);
+                        useCache(DefaultCacheCoordinator::noOp);
```

Comment:
💅 this can be inlined as `() -> {}`. A static `noOp` method confuses the readers of the class, IMO.